redbadger/website-honestly
# Track footer links with ga

### Motivation

- [Related story](https://trello.com/c/x0bhU1xJ/62-add-event-tracking-code-to-navigation-in-footer)

This PR now moves away from the analytics.js script to a React oriented wrapper: react-ga.
This allows us to trigger events easily within the React DOM.

Used to track the usage of links in the footer. Each click on a footer link now triggers an `event` for GA. The usual navigation GA hooks are still there.

### Test plan

1. Push to production, click the footer links.
2. Document the order in which you click the links in footer. (may be useful if anything goes wrong)
3. Check current GA navigations are still being flagged.
4. Wait 24 hours. GA doesn't publish these events in complete real time (I _think_), so you might have to wait 24 hours for the results to show up.

Other edge cases to note:

- Need to make sure the event is fired correctly when navigating to the blog from the footer. The [Google Tag Assistant](https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk?hl=en) plugin for chrome is very handy for testing this.

### Pre-merge checklist
